src/Tmds.DBus.Tool/Tmds.DBus.Tool.csproj

```diff
@@ -2,7 +2,9 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFrameworks>netcoreapp2.1;netcoreapp3.0</TargetFrameworks>
+    <PackAsTool>true</PackAsTool>
```

Comment:
Ah, this turns this into a global tool.

I have some questions about this. I'll create an issue and CC you on it.